docs/source/tutorial/translational_toy_example.rst

```diff
@@ -23,8 +23,8 @@ be able to exactly learn the geometric structure.
     from pykeen.pipeline import pipeline
     tf = ...
     results = pipeline(
-        training_triples_factory=tf,
-        testing_triples_factory=...,
+        training=tf,
+        testing=...,
```

Comment:
@cthoyt is there a (easy) way to automatically check that the code examples are syntactically correct with the current signatures?

Reply:
I've looked into doing this on RST files... it doesn't work unless you use the >>> python syntax and that's super messy :/